platforms/saturn.py

```diff
@@ -0,0 +1,148 @@
+from litex.build.generic_platform import *
+from litex.build.xilinx import XilinxPlatform
+
+#this is for the LX45 version
+#check on "no connect pins" for LX9 and LX25
+
+_io = [
+    ("clk100", 0, Pins("V10"), IOStandard("LVTTL")),
+
+    ("serial", 0,
+        Subsignal("tx",  Pins("L18")),
+        Subsignal("rx",  Pins("L17"), Misc("PULLUP")),
+        Subsignal("cts", Pins("M16"), Misc("PULLUP")),
+        Subsignal("rts", Pins("M18"), Misc("PULLUP")),
+        IOStandard("LVTTL")
+    ),
+
+    ("usb_fifo", 0,
```

Comment:
What type of USB FIFO is on this board?

Reply:
It's the FT2232H - one channel for JTAG, the other as FIFO
    Should there be a note in the platform file?

Reply:
Yes, that would be good!